src/c++/perf_analyzer/client_backend/openai/openai_client.cc

```diff
@@ -63,6 +63,14 @@ namespace openai {
 void
 ChatCompletionRequest::SendResponse(bool is_final, bool is_null)
 {
+  // if final response has already been sent
```

Comment:
The classes in this file should be renamed since they aren't specific to Chat Completions.